src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -74,6 +82,28 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
         }, 500);
     }
 
+    loadFirstMessage(): void {
+        const firstMessageContent = {
+            textContent: 'artemisApp.exerciseChatbot.firstMessage',
+            type: IrisMessageContentType.TEXT,
+        } as IrisMessageContent;
+
+        const firstMessage = {
+            sender: IrisSender.LLM,
+            id: 0,
+            content: [firstMessageContent],
+            sentAt: dayjs(),
+        } as IrisServerMessage;
+
+        if (this.messages.length === 0) {
+            this.isFirstMessage = true;
+            this.stateStore.dispatch(new ActiveConversationMessageLoadedAction(firstMessage));
+        } else if (this.messages[0].id === 0) {
```

Comment:
Btw why do you need this case distinction?
else if (this.messages[0].id === 0)

Reply:
Because if we close the chat window but don't reload the page, the first message can not be displayed in a right way (in this case, the isFirstMessage is false)

Reply:
But then we need to make sure the server does not send us messages with an id 0